### ISSUE TITLE
consider tagging a new git release

### ISSUE THREAD

**Issue by anthraxx** [ambrop72/badvpn#89]
Hi, it would be awesome if you could tag a new git release. The last one is from Apr 9, 2015 and there have been quite some commits in the meanwhile. Would be awesome to make distros pick up all those enhancements.

**Comment by ambrop72** [comment]
Good idea, I'll merge the UDP SOCKS feature and probably update lwip, then make a new release. Should be done in a couple of days.

**Comment by yzou** [comment]
> Good idea, I'll merge the UDP SOCKS feature and probably update lwip, then make a new release. Should be done in a couple of days.

How is it going with UDP SOCKS feature?

**Comment by MDagni** [comment]
I have just found this project and would like to use it in my Linux VM, but the date of the last release makes me think... Doesn't anybody care about this project anymore??

**Comment by anthraxx** [comment]
> I have just found this project and would like to use it in my Linux VM, but the date of the last release makes me think... Doesn't anybody care about this project anymore??

Well there seem to be some commits happening. But it unfortunately looks like nobody looks at the issues ... at least this issue os open for nearly 2 years now and no tag was created. So no distro package :crying_cat_face:

**Comment by ambrop72** [comment]
Hey,
I'm not available to do much work on this project lately. I may be able to make a new release in a few days.

**Comment by MDagni** [comment]
Hi @ambrop72 . Thank you for your answer.
Can we use it with the latest release, or should we use the master branch?

**Comment by anthraxx** [comment]
> Hi @ambrop72 . Thank you for your answer.
> Can we use it with the latest release, or should we use the master branch?

I'm a distro packager, using latest master branch is not a useful option. I would really need that new release tag :cat:
